<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>81</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="red1_5">
       <property name="text">
        <string>Led1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="green1_5">
       <property name="text">
        <string>Led2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yellow1_5">
       <property name="text">
        <string>Led3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="rail005">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>79</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Led3</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>81</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="red1_6">
       <property name="text">
        <string>Led1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="green1_6">
       <property name="text">
        <string>Led2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yellow1_6">
       <property name="text">
        <string>Led3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>81</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="red1_3">
       <property name="text">
        <string>Led1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="green1_3">
       <property name="text">
        <string>Led2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yellow1_3">
       <property name="text">
        <string>Led3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>81</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="red1">
       <property name="text">
        <string>Led1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="green1">
       <property name="text">
        <string>Led2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yellow1">
       <property name="text">
        <string>Led3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="rail004">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>79</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Led3</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_status">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>001</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>002</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>003</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>410</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>006</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="rail002">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>79</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Led4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>004</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="rail001">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>79</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Led4</string>
    </property>
   </widget>
   <widget class="QLabel" name="rail006">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>240</y>
      <width>79</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Led3</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>81</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="red1_4">
       <property name="text">
        <string>Led1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="green1_4">
       <property name="text">
        <string>Led2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yellow1_4">
       <property name="text">
        <string>Led3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>621</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>红</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>绿</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>黄</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="cmd_port_name">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="currentText">
     <string>COM1</string>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="rail003">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>79</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Led3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>005</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>81</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="red1_2">
       <property name="text">
        <string>Led1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="green1_2">
       <property name="text">
        <string>Led2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yellow1_2">
       <property name="text">
        <string>Led3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始演示</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>470</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
